solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -106,7 +106,7 @@ function update_solution_config(){
         yq e -i '.solution.common.s3.default_iam_users.auth_user = "user_name"' solution.yaml
         yq e -i '.solution.common.s3.max_start_timeout = 240' solution.yaml
         yq e -i '.solution.common.s3.extra_configuration = ""' solution.yaml
-        yq e -i '.solution.common.motr.num_client_inst = 0' solution.yaml
+        yq e -i '.solution.common.motr.num_client_inst = 1' solution.yaml   
```

Comment:
Do we need to set this to `1` always or only when it's data pod only deployment

Reply:
Yes, only with data pod deployment.

Reply:
Should this be conditional then?

Reply:
@shailesh-vaidya, added condition and tested both deployment scenarios.
https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/gc-space/job/test-setup-cortx-rgw-cluster/79/
https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/gc-space/job/test-setup-cortx-rgw-cluster/80/